src/rest_framework_jwt/blacklist/views.py

```diff
@@ -0,0 +1,37 @@
+from rest_framework import status
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+from .serializers import BlacklistTokenSerializer
+from ..authentication import JSONWebTokenAuthentication
+
+
+class BlacklistView(ModelViewSet):
+    permission_classes = (IsAuthenticated, )
+    authentication_classes = (JSONWebTokenAuthentication, )
+    queryset = BlacklistedToken.objects.all().order_by('-blacklisted_at')
+    serializer_class = BlacklistTokenSerializer
+
+    def create(self, request, *args, **kwargs):
+        serializer = self.get_serializer(data=request.data)
+        if not serializer.is_valid():
+            return Response(
+                serializer.errors, status=status.HTTP_400_BAD_REQUEST
+            )
+
+        token = serializer.data.get('token')
+        token_payload = JSONWebTokenAuthentication.jwt_decode_token(token)
+        token_username = JSONWebTokenAuthentication.jwt_get_username_from_payload(token_payload)
+
+        if token_username != request.user.username:
+            return Response(status=status.HTTP_403_FORBIDDEN)
+
+        blacklisted_token = BlacklistedToken.objects.get_or_create(token=token)[0]
```

Comment:
1. When using `get_or_create`, if you don't need the `created` value, a common workaround is to assign it to [an underscore "throwaway" variable](https://stackoverflow.com/a/5893946): `_`. This indicates that this value is intentionally ignored.

2. This statement is susceptible to race conditions (as mentioned in [`models.py`](https://github.com/Styria-Digital/django-rest-framework-jwt/pull/27/files#diff-a7aff25f8face4166d47a31b3845918bR5)).